Hai Legacy: tweak according to player feedback (#11097)
endless-sky/endless-sky: data/hai/unfettered side missions.txt

	invisible
	shipyard
	source "Hai-home"

> 	destination "Darkcloak"

Reviewer: why is there a destination for this mission? Since its actually hai legacy 2 that does the actual traveling?

Author: If you don't have a destination it'll default to the place it originated from and remain active for no good reason (or I'd have to fail it)

Reviewer 2: Yeah, it seems like this should just have `fail` in the `on accept`. You never check for `"Hai Legacy 1: done", so might as well just clean up the mission immediately. Unless it's the case that the clearance line below this is important to have, in which case this is actually fine.

					decline
			label first
			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull on your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ships, as if to mimic the actions of a regular passerby, they address you again.`
			`	"My associates and I have a lucrative business proposition for you. It will require your undivided attention, so you would do well to finish any ongoing business first. If you are interested, come find us at these coordinates."`

> 			`	"We note you operate multiple ships, however, this is a solitary affair; make sure you can park most of your ships."`

Reviewer 3: ```suggestion
			`	"We note you operate multiple ships. However, this is a solitary affair; make sure you can park most of your ships."`
```

> 			`	"We note you operate multiple ships. However, this is a solitary affair; make sure you can park most of your ships."`

Reviewer 2: This second line only display if you have more than one ship, but if it does display then the previous line shouldn't end in its quotation mark. Instead of adding another `to display`, you should start this second line with something that isn't a quotation so that the previous line is correct regardless of which line comes next.

> 			`	Glancing over at your fleet on the nearby landing pad, they add, "It's a solitary affair -- make sure you can park most of your ships."`

Reviewer 4: I don't think this works.
From the spaceport description of Hai-home:
https://github.com/endless-sky/endless-sky/blob/20bcaae66d2b7b0f0217ec6f13977d9c40b3fe79/data/map%20planets.txt#L2373
Maybe if I were browsing the shipyard from some sort of computer terminal in the hangar bay where I parked my ships, but then why would the fellow tell me to "keep walking" or point to ships in the shipyard, and why would there be a crowd of passers by?
How about:
```suggestion
			`	"We note you operate multiple ships, however, this is a solitary affair; make sure you can park most of your ships."`
```
Or similar.

				to display

> 					has "total ships" > 1

Reviewer 5: ```suggestion
					"total ships" > 1
```

					decline
			label first
			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull on your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ships, as if to mimic the actions of a regular passerby, they address you again.`

> 			`	"My associates and I have a lucrative business proposition for you. It will require your undivided attention, so you would do well to finish any ongoing business first. If you are interested, come find us at these coordinates." Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location, and if you cross-reference it with the local maps, it appears to point towards a beach on the other side of the planet.`

Reviewer 6: Had to break up the paragraph to get the "to display" in without touching the rest. I don't think that disrupts the flow, but if you disagree -- alternatively I could move it to the bottom and have them just pop back out of the crowd for a moment to tell you, or make a whole branch bit with both versions as a single paragraph.

```suggestion
			`	"My associates and I have a lucrative business proposition for you. It will require your undivided attention, so you would do well to finish any ongoing business first. If you are interested, come find us at these coordinates." 
			`	Glancing over at your fleet on the nearby landing pad, they add, "It's a solitary affair -- make sure you can park most of your ships."`
				to display
					has "total ships" > 1
			`	Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location, and if you cross-reference it with the local maps, it appears to point towards a beach on the other side of the planet.`
```

Reviewer 5: In the first paragraph, the suggestion is missing a backtick.

Reviewer 6: Good catch, now fixed.

				accept
			label fleeing
			action
				fail
			`The Unfettered don't seem too pleased to see you leaving with their prize. The challenger is already waiting for you in orbit, and hails you. "Where do you think you're going, monkey?"`
				decline
			label "too many ships"
			`As you take off to join your fleet in orbit, you receive a hail from the Unfettered. "You must leave your fleet on hold here. If they help you, the duel will be forfeit!"`
				decline
			label anomaly
			`You are hailed by the Unfettered as soon as you reach orbit. "You think us blind, monkey? That is not the relic! You're not even worth a launch sequence."`
			action
				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
				fail
	on enter "Ehma Ti"
		conversation
			branch "too many ships"
				"total ships" > 1
			`As you enter the system you can see your challenger is indeed waiting for you, and they promptly send you a hail. "Show us what you've got, monkey! Show us that you control your weapons well, as any warrior should! Prove to us that you have the strength to disable our ships, yet still have the restraint not to destroy them!"`
				accept
			label "too many ships"

> 			`As you take off to join your fleet in orbit, you receive a hail from the Unfettered. "If you can't even count how many ships you have, you're not worthy of our time, monkey! Dishonor will get you nowhere with us True Hai!"`

Reviewer 7: You aren’t taking off anymore so this should probably be changed

				"total ships" > 1
			branch anomaly
				not "flagship model: Anomalocaris"

> 			`As you leave the planet's atmosphere, you receive a short message. "We are waiting for you in Ehma Ti today, do not disappoint us."`

Reviewer 3: ```suggestion
			`As you leave the planet's atmosphere, you receive a short message. "We are waiting for you in Ehma Ti today. Do not disappoint us."`
```
